contracts/priv/unity.yml

```diff
@@ -1526,6 +1526,9 @@ components:
         isOperator:
           type: boolean
           description: whether the user is an operator
+        operatorRole:
+          type: string
```

Comment:
can we make this into an enum?

Reply:
good call. i made it an interface(enum like) in quartz. missed here. doing that.

Reply:
@Asalem1 done!